From d42cc843abe79859f15a2da9541ca7336d3551db Mon Sep 17 00:00:00 2001
From: Jason Rogena <j.rogena@cgiar.org>
Date: Thu, 27 Mar 2014 11:41:02 +0300
Subject: [PATCH 131/213] Version 0.8 Revision 6: Text from servicing bull name
 AutoCompleteTextView now being parsed to bullEarTagNo in json when sending
 json in AddCowEvent activity

---
 .../src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java   | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
index 615a795..6cd5af4 100644
--- a/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
+++ b/MistroFarmer/src/main/java/org/cgiar/ilri/mistro/farmer/AddEventActivity.java
@@ -505,7 +505,8 @@ public class AddEventActivity extends SherlockActivity implements View.OnClickLi
                 jsonObject.put("remarks", remarksET.getText().toString());
                 jsonObject.put("strawNumber", strawNumberET.getText().toString());
                 jsonObject.put("vetUsed", vetUsedET.getText().toString());
-                jsonObject.put("bullName", bullNameACTV.getText().toString());
+                //jsonObject.put("bullName", bullNameACTV.getText().toString());
+                jsonObject.put("bullEarTagNo", bullNameACTV.getText().toString());
                 //jsonObject.put("bullEarTagNo", bullETNACTV.getText().toString());
                 jsonObject.put("noOfServicingDays", noOfServicingDaysET.getText().toString());
                 if(servicingIDs != null) {
-- 
2.7.1

